WinImageBuilder.psm1

```diff
@@ -293,6 +314,9 @@ function Check-DismVersionForImage {
         (Get-Command dism.exe).FileVersionInfo.ProductVersion
     if ($image.ImageVersion.CompareTo($dismVersion) -gt 0) {
         Write-Warning "The installed version of DISM is older than the Windows image"

```

Comment:
Write-Warning "The installed version of DISM is different than the Windows image's"